doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -10,6 +10,6 @@
         "#1": "Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
         "#2": "For a sqlite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
         "#3": "Set \"serverVersion\" to your server version to avoid edge-case exceptions and extra database calls",
-        "DATABASE_URL": "mysql://root@127.0.0.1:3306/symfony?charset=utf8mb4&serverVersion=5.7"
+        "DATABASE_URL": "mysql://app:app@127.0.0.1:3306/app?charset=utf8mb4&serverVersion=5.7"
```

Comment:
I also think it's a good practice to propose users a non-root user name and a non-empty password. It encourages best practices at first, and next, it helps when dealing with PaaS where environment can define very long & complex database names automatically (like JawsDB heroku add-on for example).

Moreover, it explicitly adds verbosity to the full database url that will be parsed by the DBAL.

I'd go further by replacing it with more verbose names:

`mysql://db_user:db_password@127.0.0.1:3306/app?charset=utf8mb4&serverVersion=5.7`